content/md/products-super-power/work-on-your-assets.md

```diff
@@ -32,6 +33,24 @@ You can find a **locale and channel switcher** on each asset, meaning that if yo
 
 ![Assets Locale Switcher](../img/Assets_LocaleSwitcher.png)
 
+## Bulk edit a selection of asset attributes
+It is possible to bulk edit your own selection of asset attributes within an asset family.
+Let's imagine that you need to add the same tag to several assets, for instance, their main color attribute. Instead of manually adding the tag for each one of these assets, you can easily do it once for your selection of assets, thanks to a bulk action. Here's how to do do:
+
+1. Check the box under each asset you would like to bulk edit
+1. In the bottom bar, click on `edit`
+1. At the "edit" step, click on `add attributes` in the top right menu
+![Assets attributes selection](../img/Assets_bulk-edit-add-attributes.png)
+1. Select the attributes you would like to bulk edit
+![Assets attributes selection](../img/Assets_bulk-edit-attributes-selection.png)
+1. Fill in the values you want for each selected attribute
+1. Once you are done, click on `next` in the top right corner
+1. At the "confirm" step, you can visualise the changes that will be made. If you want to continue, click on `Confirm`
+
+::: warning
+Please note that it is possible to add the same attribute multiple times. For instance, if you would like to edit an attribute that is scopable and/or localisable, you will have the possibility to do so by adding the attribute as many times as you want, but you will have to carefully select a different locale and/or scope for each of them so that you will avoid to setting different values for the same attribute with the same locale and/or same scope.
+:::
+
```

Comment:
Maybe add permission requirement for mass edit asset

Reply:
Thanks for the clarification!